posts/2020-10-23-beta-20.0.0.12.adoc

```diff
@@ -0,0 +1,452 @@
+---
+layout: post
+title: "MicroProfile Updates, New Jakarta EE 9 Packages and more in Open Liberty 20.0.0.12-beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/austin0
+author_github: https://github.com/austin0
+seo-title: MicroProfile Updates, New Jakarta EE 9 Packages and more in Open Liberty 20.0.0.12-beta - OpenLiberty.io
+seo-description: OpenLiberty updates for a number of commonly used MicroProfile features and four new Jakarta EE 9 packages, now available in Open Liberty 20.0.0.12-beta.
+blog_description: Updates for a number of commonly used MicroProfile features and four new Jakarta EE 9 packages.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile Updates, New Jakarta EE 9 Packages and more in Open Liberty 20.0.0.12-beta
```

Comment:
Change title to "More MicroProfile 4.0 previews, MicroProfile LRA, and more Jakarta EE 9 features in Open Liberty 20.0.0.12-beta".

Also change it in `title` and `seo-title` in the front matter/metadata.

Reply:
Change `seo-description` and `description` to:

"Get previews of more MicroProfile 4.0 features (Config 2.0 and JWT 1.2), MicroProfile LRA, support for Java SE 15, and more Jakarta EE 9 features."